findex_sqlite/src/ser_de.rs

```diff
@@ -56,7 +53,7 @@ where
     }
 
     fn read(_de: &mut Deserializer) -> Result<Self, Self::Error> {
-        Err(FindexErr::Other(
+        Err(Error::CryptoError(
```

Comment:
Is it a crypto error here? Doesn't look like one…

Reply:
Indeed, a `CoversionError` seems more appropriate, thanks

Reply:
Done in 578e60e